db/migrate/20150316140954_create_versions.rb

```diff
@@ -0,0 +1,13 @@
+class CreateVersions < ActiveRecord::Migration
+  def change
+    create_table :versions do |t|
+      t.string   :item_type, :null => false
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Put one space between the method name and the first argument.